Assets/Scripts/UI/MainMenu/MainMenuController.cs

```diff
@@ -111,10 +119,20 @@ private void Start()
             mainMenuCamera.SetActive(true);
             // Reset loading screen
             LoadingScreen.ResetCounter();
+
+
+            if (firstInputJoined.IsMouseAndKeyboard) EnableVisibleMouse();
```

Comment:
Basically the same code in EnableVisibleMouse as in the `else` branch, consider making the `else` branch call its own method